pkg/kf/quotas/kfquota.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quotas
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// KfQuota provides a facade around v1.ResourceQuota for accessing and mutating its
+// values.
+type KfQuota v1.ResourceQuota
+
+// GetName retrieves the name of the quota.
+func (k *KfQuota) GetName() string {
+	return k.Name
+}
+
+// SetName sets the name of the quota.
+func (k *KfQuota) SetName(name string) {
+	k.Name = name
+}
+
+// TODO: set and get actual resource quota fields
+func (k *KfQuota) GetMemory() resource.Quantity {
```

Comment:
nit: these public functions should have docstrings

Reply:
done